### DISCUSSION TITLE
Issues moving ./storage directory

### DISCUSSION THREAD

**Discussion by lennartbrandin** [photoprism/photoprism#2656] [Q&A] [answered]
I copied the storage to my NAS, modified the docker-compose.yml and the thumbnails are not working correctly anymore.

#### Issue

![image](https://user-images.githubusercontent.com/92579517/187077270-df40ef84-126d-497f-82fa-74278523fb0b.png)

Log contains plenty of those three:
```
level=error msg="thumb: not cached"
level=error msg="thumb: file abc/xyz.AVI.jpg is missing" 
level=warning msg="thumb: not found"
```

#### Config & co

**Folder permissions**
(Top one is the new location on the NAS and bottom the initial one, which is owned by root for some reason. The container is not running with root permissions.)
![image](https://user-images.githubusercontent.com/92579517/187075607-67dee873-e752-4933-be5c-92127a37137f.png)

**Docker-compose.yml**
![image](https://user-images.githubusercontent.com/92579517/187077525-fc4241c2-0ca2-4e7f-b030-b1d907c6ede6.png)

**Comment by graciousgrey** [comment, marked as answer]
You should only change the left part of this line in your docker-compose.yml

`- "./storage:/photorprism/storage"`

./storage is the location of the storage folder on the server where you have photoprism installed.
photoprism/storage is the location within the container
